h2o-algos/src/test/java/hex/glm/GLMPValuesTest.java

```diff
@@ -0,0 +1,213 @@
+    System.out.println("pValuesManual: " + Arrays.stream(pValuesManual)
+            .mapToObj(d -> String.format("%.6f", d)).collect(Collectors.joining(", ")));
+    System.out.println("pValues:       " + Arrays.stream(pValues)
+            .mapToObj(d -> String.format("%.6f", d)).collect(Collectors.joining(", ")));
+    assertArrayEquals(zValuesManual, zValues, tot1);
+    assertArrayEquals(manualStdErr, stdErr, tot2);
+    assertArrayEquals(pValuesManual, pValues, tot3);
+  }
+
+  /**
+   * Assert z-values, standard error and p-values for the given submodel.
+   *
+   * @param submodel                 trained submodel
+   * @param residualDegreesOfFreedom residual degrees of freedom for the model
+   * @param manualStdErr             manually computed standard error
+   * @param reg
+   * @param tot1                     delta for z-values
+   * @param tot2                     delta for standard errors
+   * @param tot3                     delta for p-values
+   */
+  public static void assertStandardErrForSubmodels(GLMModel.Submodel submodel, long residualDegreesOfFreedom,
+                                                   double[] manualStdErr, double reg, double tot1, double tot2, double tot3) {
+    double[] zValues = submodel.zValues();
+    double[] stdErr = submodel.stdErr();
+    double[] pValues = submodel.pValues(residualDegreesOfFreedom);
+    double[] zValuesManual = genZValues(submodel.beta, manualStdErr, reg);
+    double[] pValuesManual = genPValues(zValuesManual, submodel.dispersionEstimated, residualDegreesOfFreedom);
+    System.out.println("pValuesManual: " +
+            Arrays.stream(pValuesManual).mapToObj(d -> String.format("%.6f", d)).collect(Collectors.joining(", ")));
+    System.out.println("pValues:       " +
+            Arrays.stream(pValues).mapToObj(d -> String.format("%.6f", d)).collect(Collectors.joining(", ")));
+    assertArrayEquals(zValuesManual, zValues, tot1);
+    assertArrayEquals(manualStdErr, stdErr, tot2);
+    assertArrayEquals(pValuesManual, pValues, tot3);
+  }
+
+  /**
+   * Manually generate p-values from given z-values.
+   *
+   * @param zValues                  z-values
+   * @param dispersionEstimated      flag for selecting the distribution
+   * @param residualDegreesOfFreedom degrees of freedom if the dispersionEstimated is true
+   * @return list of p-values
+   */
+  public static double[] genPValues(double[] zValues, boolean dispersionEstimated, long residualDegreesOfFreedom) {
+    RealDistribution gaussian = dispersionEstimated
+            ? new TDistribution(residualDegreesOfFreedom)
+            : new NormalDistribution();
+    double[] pValues = new double[zValues.length];
+    for (int index = 0; index < zValues.length; index++)
+      pValues[index] = 2 * gaussian.cumulativeProbability(-Math.abs(zValues[index]));
+    return pValues;
+  }
+
+
+  /**
+   * Setup before test
+   */
+  @BeforeClass
+  public static void setup() {
+    stall_till_cloudsize(1);
+  }
```

Comment:
Please use this one for your brand you test:
@RunWith(H2ORunner.class)
@CloudSize(1)

Put this right before you define your test class.